### PR TITLE
Dop 41

### PR DESCRIPTION
**Pull request by CosminBriscaru**
Description

Adding Anthos and Istio related resources.
Jira -> https://coremaker.atlassian.net/browse/DOP-41

Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ X ] Feature (non-breaking change which adds functionality)
- [ X ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring (no functional changes, formatting, renaming)
- [ ] Other (please describe):

TODOs

- [ X ] Ensure the branch is named correctly with the ticket number. (eg: DOP-2222)
- [ ] Update the docs.
- [ X ] Run the pre-commit checks successfully.

Other information

Adding reviewers:
@coremaker/devops

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -3,6 +3,12 @@ variable "k8s_namespaces" {
     name          = string
     has_public_ip = bool
     dns_records   = set(string)
+    // labels = list(object({
```
Comment (strediecosmin) [inline variables.tf]: I think we can remove this comments

**Review by strediecosmin — CHANGES_REQUESTED** [review]

File: helm-charts/anthos-cpr/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name: anthos-control-plane-revision
+version: v0.0.1
+description: A Helm chart for the Anthos Control Plane Revision.
+appVersion: v1.4.3
+maintainers:
+  - name: coremaker
+    email: it-infra@coremaker.io
```
Comment (mihstaub) [inline helm-charts/anthos-cpr/Chart.yaml:7]: Maybe we should change this to devops@coremaker.io.

Comment (mihneastaub) [inline helm-charts/anthos-cpr/Chart.yaml]: Maybe we should change this to: devops@coremaker.io

Comment (CosminBriscaru) [reply helm-charts/anthos-cpr/Chart.yaml]: Not sure if we have that one but makes sense anyway

**Review by strediecosmin — APPROVED** [review]
LGTM!

**Pull request merged**